packages/timeline-state-resolver-types/src/integrations/obs.ts

```diff
@@ -1,35 +1,43 @@
 import { DeviceType } from '..'
 
 export enum OBSRequest {
```

Comment:
This enum is only ever used in TSR, not externally. So we could move it into the TSR package instead.

(
While we're at it, add this comment as well:
`/** values are derived from keyof OBSRequestTypes in obs-websocket-js */`
)